clap_complete/src/dynamic/completer.rs

```diff
@@ -421,16 +393,7 @@ fn complete_arg_value(
         values = values
             .into_iter()
             .map(|comp| {
-                CompletionCandidate::new(format!(
-                    "{}{}{}",
-                    prefix,
-                    arg.get_value_delimiter()
-                        .map(|c| c.to_string())
-                        .unwrap_or("".to_string()),
-                    comp.get_content().to_string_lossy()
-                ))
-                .help(comp.get_help().cloned())
-                .visible(comp.is_visible())
+                comp.add_prefix(format!("{}{}", prefix, arg.get_value_delimiter().unwrap()))
```

Comment:
Please use `expect` to document the reason for an unwrap (and the old code didn't unwrap)